galaxy_wrappers/NSPDK/NSPDK_candidateClusters.pl

```diff
@@ -35,7 +35,8 @@
     $CI = 1;
 }
 
-system("cat $SVECTOR_DIR/* > $SVECTOR_DIR/data.svector");
+system("cd $SVECTOR_DIR && ls *.group.gspan | sort -n -t . -k 1 | xargs cat > data.svector ");
```

Comment:
Please test this alternative :`sort -n -t . -k 1` ->  `sort -n -t'.' -k 1`
One has to be careful and try using quotes specially when deal with special characters like dot